adm_program/modules/dates/dates.php

```diff
@@ -498,7 +510,7 @@
             {
                 if($outputNumberMembers > 0 || $outputNumberLeaders > 0)
                 {
-                    $buttonURL = ADMIDIO_URL.FOLDER_MODULES.'/lists/lists_show.php?mode=html&amp;rol_ids='.$date->getValue('dat_rol_id');
+                    $buttonURL = ADMIDIO_URL.FOLDER_MODULES.'/lists/lists_show.php?mode=html&amp;rol_ids='.$date->getValue('dat_rol_id').'&amp;lst_id='.$gPreferences['dates_default_list_configuration'];
```

Comment:
We should not set the default list as script parameter. The list_show should check if it's an event role and then set the default list. So it's also possible to set a specific list for an special event. Therefore enhance the method `getDefaultList()` of TableRoles. Should I do this?